src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -190,6 +193,67 @@ public function table(array $headers, array $rows)
         $this->newLine();
     }
 
+    /**
+     * Formats a horizontal table.
+     */
+    public function horizontalTable(array $headers, array $rows)
+    {
+        $style = clone Table::getStyleDefinition('symfony-style-guide');
+        $style->setCellHeaderFormat('<info>%s</info>');
+
+        $table = new Table($this);
+        $table->setHeaders($headers);
+        $table->setRows($rows);
+        $table->setStyle($style);
+        $table->setHorizontal(true);
+
+        $table->render();
+        $this->newLine();
+    }
+
+    /**
+     * Formats a list of key/value horizontally.
+     *
+     * Each row can be one of:
+     * * 'A title'
+     * * ['key' => 'value']
+     * * new TableSeparator()
+     */
```

Comment:
an `@param` could be `string|array|TableSeparator $list` isnt it?

Reply:
`@param string|array|TableSeparator ...$list` is the correct one for variadics

Reply:
Added.